modules/monitoring-resolving-the-kubepersistentvolumefillingup-alarm-firing-for-prometheus.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * monitoring/troubleshooting-monitoring-issues.adoc
+// * support/troubleshooting/investigating-monitoring-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resolving-the-kubepersistentvolumefillingup-alarm-firing-for-prometheus_{context}"]
+= Resolving the KubePersistentVolumeFillingUp alarm firing for Prometheus
```

Comment:
The [KCS article](https://access.redhat.com/solutions/7033177) is specifically for the case of the `prometheus-k8s-*` pod. Is this the main user case we want to address, or do we also want to include the general recommendations on what to do if the `KubePersistentVolumeFillingUp` alarm starts firing for other system components as described here: https://github.com/openshift/runbooks/blob/master/alerts/cluster-monitoring-operator/KubePersistentVolumeFillingUp.md?